app/views/users/index.html.erb

```diff
@@ -1,11 +1,14 @@
-<h1>User pages </h1>
-<% @user.each do |item| %>
-   <div>
-    <img src="<%= item.photo %>" alt="user-image">
-    <span>
-     <h3><%= item.name %></h3>
-     <p>Number of post: <%= item.posts_counter %> </p>
-    </span>
-   </div> 
-<% end %> 
+<div class = "container">
 
+<% @user.each do |p| %>
+<div class = "user-row">
+    <div class = "photo">
+        <%= image_tag p.photo %>
+    </div>
+    <div class = "intro-details">
+       <p class = "user-name"> <%= p.name  %> </p>
+       <p class = "post-counter">Number of posts:  <%= p.posts_counter %> </p>
+    </div>    
+</div>
```

Comment:
- [ ] I am unable to go from one page to another. Please make sure to make a link so that one can go from one page to another.

Reply:
I have fix it